doc/t265.md

```diff
@@ -0,0 +1,107 @@
+
+```cpp
+rs2::pipeline pipe;
+
+rs2::config cfg;
+cfg.enable_stream(RS2_STREAM_GYRO);
+cfg.enable_stream(RS2_STREAM_ACCEL);
+cfg.enable_stream(RS2_STREAM_POSE);
+
+pipe.start(cfg);
+
+while (app) // Application still alive?
+{
+    rs2::frameset frameset = pipe.wait_for_frames();
+
+    // Find and retrieve IMU and/or tracking data
+    if (rs2::motion_frame accel_frame = frameset.first_or_default(RS2_STREAM_ACCEL))
+    {
+        rs2_vector accel_sample = accel_frame.get_motion_data();
+        //std::cout << "Accel:" << accel_sample.x << ", " << accel_sample.y << ", " << accel_sample.z << std::endl;
+        //...
+    }
+
+    if (rs2::motion_frame gyro_frame = frameset.first_or_default(RS2_STREAM_GYRO))
+    {
+        rs2_vector gyro_sample = gyro_frame.get_motion_data();
+        //std::cout << "Gyro:" << gyro_sample.x << ", " << gyro_sample.y << ", " << gyro_sample.z << std::endl;
+        //...
+    }
+
+    if (rs2::pose_frame pose_frame = frameset.first_or_default(RS2_STREAM_POSE))
+    {
+        rs2_pose pose_sample = pose_frame.get_pose_data();
+        //std::cout << "Pose:" << pose_sample.translation.x << ", " << pose_sample.translation.y << ", " << pose_sample.translation.z << std::endl;
+        //...
+    }
+}
+```
+
+## FAQ
+
+### Is this a depth camera?
+No, the T265 does not offer any Depth information. It can provide position and orientation (6-DOF), IMU (accelerometer and gyroscope), and two monochrome fisheye streams.
+
+### Can I use it together with a depth camera?
+Yes, this device can be used with RealSense (and most likely any 3rd-party) depth cameras. It is passive and comes with an IR-cut filter.
+
+### What platforms are supported?
+At launch the T265 will be supported via librealsense on Windows and Linux, and via ros-realsense. Mac-OS and Android support are planned but not scheduled yet, as well as OpenVR integration.
+
+### What are the system requirements?
+Any board with USB2 capable of receiving pose data at 260 times a second should be sufficient. The device was validated on Intel NUC platform. To access the fisheye streams, USB3 is required.
+
+### Will the same SLAM be available for the D435i as a software package?
+We are still looking into it, please stay tuned.
+
+### What is wheel odometry and how does it help T265 navigate?
+
+Wheel odometry is the use of sensors to measure how much a wheel turns, it can be used to estimate changes in a wheeled robots position. T265 has wheel odometry support built in, allowing it to use the data from these sensors to refine the position data of the robot. Providing robotic wheel odometer or velocimeter data over USB to TM2 will make the tracking much more robust on wheeled robots, which otherwise can experience many tracking failures. We consider odometer input to be a requirement for robust tracking on wheeled robots.
```

Comment:
Should be T265 instead of TM2 in "Providing robotic wheel odometer or velocimeter data over USB to TM2 will make the tracking..."